test/integration/data-types/geometry.test.ts

```diff
@@ -86,25 +85,6 @@ describe(getTestDialectTeaser('DataTypes'), () => {
     });
 
     it('correctly parses an empty GEOMETRY field', async () => {
```

Comment:
Okay, this test seems to be broken.

---

For `mariadb` it indeed returns `null`.

For `mysql` it returns `Invalid GIS data provided to function st_geometryfromtext.`

For `postgres oldest` it returns `Too few ordinates in GeoJSON`

For `postgres latest` it returns the following;
```
{
  "coordinates": [
    NaN
    NaN
  ]
  "crs": {
    "properties": {
      "name": "EPSG:4326"
    }
    "type": "name"
  }
  "type": "Point"
}
```